website/content/en/docs/upgrading-sdk-version/v1.0.0.md

```diff
@@ -0,0 +1,68 @@
+---
+title: v1.0.0
+weight: 999000000
+---
+
+## Upgrade your project from version "2" to "3-alpha"
+
+The SDK's default Go plugin no longer supports OLM- or scorecard-related project files nor writes a `plugins` PROJECT field for projects scaffolded previously with `operator-sdk init --project-version=2`, Please migrate to project version "3-alpha" for support of these features by adding the following to your `PROJECT` file:
+```yaml version: "3-alpha" # Updated from "2" projectName: <output of $(basename $(pwd))> layout: go.kubebuilder.io/v2 plugins:
+  go.sdk.operatorframework.io/v2-alpha: {}
+```
```

Comment:
Is this YAML block rendering correctly?
From the preview it looks like:
![image](https://user-images.githubusercontent.com/6660644/89825053-e23b9b80-db08-11ea-89fa-6fbe3d8f0c93.png)


Shouldn't it be:
```yaml 
version: "3-alpha" # Updated from "2" 
projectName: <output of $(basename $(pwd))> 
layout: go.kubebuilder.io/v2 plugins:
  go.sdk.operatorframework.io/v2-alpha: {}
```

/cc @estroz

Reply:
Likewise for the website docs preview:
https://deploy-preview-3713--operator-sdk.netlify.app/docs/upgrading-sdk-version/v1.0.0/#upgrade-your-project-from-version-2-to-3-alpha

![image](https://user-images.githubusercontent.com/6660644/89825368-54ac7b80-db09-11ea-9679-04c784e83eec.png)

Reply:
@hasbro17 I think I got both of them.